packages/components/Avatar/src/useAvatar.ts

```diff
@@ -38,16 +38,16 @@ export const useAvatar = (props: AvatarProps): AvatarInfo => {
 
   const showRing = active === 'active' && activeAppearance === 'ring';
   const showBadge = (!active || active === 'unset') && !!badge && !!badge.status;
-  const accessibilityText = `${name || ''}${showBadge ? `, ${badge.status}` : ''}`;
-
+  const accessibilityText = `${accessibilityLabel ? `${accessibilityLabel}` : undefined || `${name || ''}`}
+   ${showBadge ? `, ${badge.status}` : ''}${active != undefined ? active : ''} ${badge?.outOfOffice ? 'Out of Office' : ''} `;
```

Comment:
I think this is starting concatenation (sorry, my brain is slow right now), which I think wouldn't be good for localization, because it assumes the structure of what you want to read out (maybe some language doesn't like space as the delineator, etc)